tokens/manage/proxy-accounts.md

```diff
@@ -84,6 +84,23 @@ For the purposes of this guide, you will be setting up a proxy account using the
 
 ## Creating a Proxy Account {: #creating-a-proxy-account }
 
+
+### Via Apps.Moonbeam.Network {: #create-via-apps-moonbeam-network }
+
+It's easy to create a proxy account on [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}. To do so, take the following steps:
```

Comment:
Using the URL as the link text will result in some of my reports returning issues with descriptive link text. It's an accessibility thing

```suggestion
It's easy to create a proxy account on [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}. To do so, take the following steps:
```